tests/integration/convert_and_import_excel_file.t

```diff
@@ -44,16 +43,38 @@ sub fake_download_image ($) {
 	return $response;
 }
 
+my @tests = (
+	{
+		test_case => "test",
```

Comment:
It would be better to stick to "id" or "desc" because I try to use it to display messages in Test.pm.

Reply:
BTW I looked and in my PR I use id / test_case or desc, so it's ok :-)